index.html

```diff
@@ -1,43 +1,177 @@
-        <ul>
-            <li><a href="https://twitter.com"><img src="images/twitter.png" alt="twitter"></a></li>
-            <li><a href="https://Linkedin.com"><img src="images/Linkedin icon.png" alt="Linkedin"></a></li>
-            <li><a href="https://medium.com"><img src="images/medium.png" alt="medium"></a></li>
-            <li><a href="https://github.com"><img src="images/github.png" alt="github"></a></li>
-            <li><a href="https://angellist.com"><img src="images/angellist.png" alt="angellist"></a></li>
-        </ul>   
+  </article>
+</main>
+<section class="mobile-work">
+  <article class="card flex-box clear-white-background">
+    <img src="./images/project_1.png" alt="tonic project preview">
+    <h3>Tonic</h3>
+    <ul class="flex-box work-info">
+      <li>CANOPY</li>
+      <li>BACKEND DEV</li>
+      <li>2015</li>
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="flex-box mobile-work-info">
+      <li>html</li>
+      <li>css</li>
+      <li>javascript</li>
+    </ul>
+    <button class="preview_button outline">See Project</button>
+  </article>
+  <article class="card flex-box clear-white-background">
+    <img src="./images/project_2.png" alt="Multi-Post project preview">
+    <h3>Multi-Post Stories</h3>
+    <ul class="flex-box work-info">
+      <li>CANOPY</li>
+      <li>BACKEND DEV</li>
+      <li>2015</li>
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="flex-box mobile-work-info">
+      <li>html</li>
+      <li>css</li>
+      <li>javascript</li>
+    </ul>
+    <button class="preview_button outline">See Project</button>
+  </article>
+  <article class="card flex-box clear-white-background">
+    <img src="./images/project_3.png" alt="tonic project preview">
+    <h3>Tonic</h3>
+    <ul class="flex-box work-info">
+      <li>CANOPY</li>
+      <li>BACKEND DEV</li>
+      <li>2015</li>
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="flex-box mobile-work-info">
+      <li>html</li>
+      <li>css</li>
+      <li>javascript</li>
+    </ul>
+    <button class="preview_button outline">See Project</button>
+  </article>
+  <article class="card flex-box clear-white-background">
+    <img src="./images/project_4.png" alt="Multi-Post project preview">
+    <h3>Multi-Post Stories</h3>
+    <ul class="flex-box work-info">
+      <li>CANOPY</li>
+      <li>BACKEND DEV</li>
+      <li>2015</li>
+    </ul>
+    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="flex-box mobile-work-info">
+      <li>html</li>
+      <li>css</li>
+      <li>javascript</li>
+    </ul>
+    <button class="preview_button outline">See Project</button>
+  </article>
+</section>
+<section class="about clear-white-background">
+  <article>
+    <h1>About <br> Myself</h1>
```

Comment:
- [OPTIONAL] Kindly change the `<h1>` tag to any other `hx` tag (i.e 2, 3, 4, ....). Professionally speaking, you can only have one `<h1>` tag on a page. This is for semantics reason.